### PR TITLE
Add AWS costs recommendations

### PR DESCRIPTION
**Pull request by mchv**

### CODE REVIEW

File: AWS-costs.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Make sure the table capacity can handle traffic spikes. Use the time range on the graphs to see the past weeks usage.
+
+
+### Cloudwatch
+Cloudwatch metrics can become really expensive without being used at all. Apart from your app metrics, you can configure some AWS SDK to send the metrics you are using, [for instance for kinesis](https://github.com/guardian/content-api/pull/1332/files#diff-60cf2d80ad25f26151966317093f6eb0R64). 
+
+### S3
+S3 may looks cheap, but the management console does not help you to understand what is inside each bucket and the size of each own. You can follow the storage size of each of your buckets from a [cloudwatch metric](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#metrics:metricFilter=Pattern%253DAWS%252FS3) called `BucketSizeBytes`. It is good idea as well to have a look the different [storage classes](https://aws.amazon.com/s3/storage-classes/) available for your data:
+
+* Infrequent access
```
Comment (rrees) [inline AWS-costs.md]: Reads seem to generally be quite expensive so I think it's worth looking at the cost for a restore in regular S3 compared to the other options and look at how frequently you might want to do that.

**Comment by rrees** [comment]
:+1:

**Comment by DiegoVazquezNanini** [comment]
:+1:

File: AWS-costs.md (hunk 1 above)
Comment (mchv) [inline AWS-costs.md]: happy to merge a change suugesting it!

**Pull request merged**